client/src/components/testimonials/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect } from "react";
+import {
+  TestimonialsSection,
+  TestimonialsSubTitle,
+  TestimonialsTitle,
+  TestimonialsQuote,
+  TestimonialsText,
+  TestimonialsClientName,
+  TestimonialsClient,
+} from "./testimonials";
+import { Container } from "../../box/absolute-box";
+import AOS from "aos";
+import "aos/dist/aos.css";
+import TinySlider from "tiny-slider-react";
+import "tiny-slider/dist/tiny-slider.css";
```

Comment:
you can take a look on [eslint-plugin-import](https://www.npmjs.com/package/eslint-plugin-import). it can help you to make your imports cleaner.